styles/styles.css

```diff
@@ -92,14 +97,15 @@
 }
 
 body {
-  display: none;
-  margin: 0;
-  font-size: 16px;
-  line-height: 1.44;
-  -moz-osx-font-smoothing: grayscale;
-  -webkit-font-smoothing: antialiased;
-  font-family: sans-serif;
-  font-weight: 900;
+    display: none;
+    margin: 0;
+    font-size: 16px;
+    line-height: 1.44;
+    -moz-osx-font-smoothing: grayscale;
+    -webkit-font-smoothing: antialiased;
+    font-family: sans-serif;
+    font-weight: 900;
```

Comment:
Default font probably should be 400 - i realize this was in the original, just putting a note here.